src/Components/App/App.tsx

```diff
@@ -24,14 +26,24 @@ const App: React.FC = () => {
   }
 
   return (
-    <div className="backGround">
-      <NavBar retrieveBooks={retrieveBooks} />
-      <main>
-        <h1 className="project-title">A Novel Idea</h1>
-        <BookCardContainer allBooks={books} oneBook={retrieveSingleBook}/>
-          <BookDetails />
-      </main>
-    </div>
+        <div className="backGround">
+          <main>
+            <h1 className="project-title">A Novel Idea</h1>
+            <Route exact path='/'>
+              <Home />
+            </Route>
+            <Route exact path='/books'>
+              <BookCardContainer allBooks={books} oneBook={retrieveSingleBook}/>
```

Comment:
`retrieveSingleBook` sounds like a function since it starts with a verb, but `oneBook` sounds like an object. Which is it? Whichever it is, change the names so it's consistent.